rskj-core/src/main/java/co/rsk/config/ConfigLoader.java

```diff
@@ -56,13 +59,38 @@ public ConfigLoader(CliArgs<NodeCliOptions, NodeCliFlags> cliArgs) {
     }
 
     public Config getConfig() {
-        Config userConfig = getConfigFromCliArgs()
-                .withFallback(ConfigFactory.systemProperties())
-                .withFallback(ConfigFactory.systemEnvironment())
-                .withFallback(getUserCustomConfig())
-                .withFallback(getInstallerConfig());
+        Config cliConfig = getConfigFromCliArgs();
+        Config systemPropsConfig = ConfigFactory.systemProperties();
+        Config systemEnvConfig = ConfigFactory.systemEnvironment();
+        Config userCustomConfig = getUserCustomConfig();
+        Config installerConfig = getInstallerConfig();
+
+        Config userConfig = ConfigFactory.empty()
+                .withFallback(cliConfig)
+                .withFallback(systemPropsConfig)
+                .withFallback(systemEnvConfig)
+                .withFallback(userCustomConfig)
+                .withFallback(installerConfig);
         Config networkBaseConfig = getNetworkDefaultConfig(userConfig);
-        return userConfig.withFallback(networkBaseConfig);
+        Config unifiedConfig = userConfig.withFallback(networkBaseConfig);
+
+        if (unifiedConfig.getBoolean(SystemProperties.PROPERTY_BC_VERIFY)) {
+            Config expectedConfig = ConfigFactory.load(EXPECTED_RESOURCE_PATH);
+            Config actualConfig = ConfigFactory.empty()
```

Comment:
Config without `systemPropsConfig` and `systemEnvConfig` which are out of our interest

Reply:
can you add this as a comment on the code directly?